pyvista/core/dataset.py

```diff
@@ -687,10 +687,10 @@ def set_active_vectors(self, name: Optional[str], preference='point'):
             elif field == FieldAssociation.CELL:
                 ret = self.GetCellData().SetActiveVectors(name)
             else:
-                raise ValueError(f'Data field ({field}) not usable')
+                raise ValueError(f'Data field ({name}) not usable')
```

Comment:
This used to show the `FieldAssociation` enum type - I don't believe that was the intent.

Reply:
The if statement here is checking the FieldAssociation.  It would be good to have both in this error message.
```suggestion
                raise ValueError(f'Data field ({name}) with type {field} not usable')
```